### DISCUSSION TITLE
session manager keeps increasing pane count

### DISCUSSION THREAD

**Discussion by mjkpolo** [zellij-org/zellij#3587] [Q&A] [answered]
Is it normal for the pane count shown by the session manager (by doing ctrl o + w) to increase every time I open it? I noticed this doesn't happen if I close it with ctrl p + x.
<img width="630" alt="Screenshot 2024-09-03 at 11 57 54 AM" src="https://github.com/user-attachments/assets/7c7b5357-36bf-4502-a616-3e6b75560414">

**Comment by cristiand391** [comment, marked as answer]
I think it's the same bug:
https://github.com/zellij-org/zellij/issues/3409

**Reply by mjkpolo** [reply]
Indeed thank you. I guess not a major issue, kind of "memory leak" territory, but knowing I'm opening more panes bugs me lol
